huaweicloud/services/elb/resource_huaweicloud_elb_certificate.go

```diff
@@ -103,10 +103,10 @@ func ResourceCertificateV3() *schema.Resource {
 }
 
 func resourceCertificateV3Create(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
-	config := meta.(*config.Config)
-	elbClient, err := config.ElbV3Client(config.GetRegion(d))
+	cfg := meta.(*config.Config)
+	elbClient, err := cfg.ElbV3Client(cfg.GetRegion(d))
 	if err != nil {
-		return diag.Errorf("error creating elb client: %s", err)
+		return diag.Errorf("error creating ELB client: %s", err)
```

Comment:
```suggestion
		return diag.Errorf("error creating ELB(v3) client: %s", err)
```
Both v2 and v3 clients are in use, and it's better to add the version number.

Reply:
the version v2 is reserved，the version v3 is deleted